client/content/banqi.css

```
@@ -105,14 +105,33 @@ html:before {
 .dead-pieces {
   height: auto;
 }
-.dead-pieces .banqi-square{
+.dead-pieces .banqi-square {
   height:45px;
   width:45px;
   background-repeat:no-repeat;
   background-size:cover;
   margin:2px;
   display:inline-block;
 }
+.dead-pieces .banqi-square.dead::before {
+    content: "";
+    background-color: black;
+    opacity: 40%;
+    border-radius: ;
```

```suggestion
    border-radius: 50%;
```

Somehow this didn't merge correctly.